### ISSUE TITLE
Linux Compatibility

### ISSUE THREAD

**Issue by FlamenWolf** [soulsmods/ModEngine2#89]
Is there anyway to get this to work for Linux? I'm using Pop!OS currently. I want to run the Reforged mod. The exe doesn't launch with wine no matter what I do so is there another option to get the mod to work?

**Comment by Keelhauled** [comment]
It would be nice to have this working as a dinput8.dll mod again, those are easier to run on linux. I tried to add modengine2.dll and lua.dll to the Elden Mod Loader mods folder and it seems to work but i don't think it can find the config.

**Comment by Keelhauled** [comment]
Just found out about this fork that works with proton https://github.com/Cloudef/ModEngine2/releases/tag/2.0.0.1-proton-v3. Hopefully it can be merged.

**Comment by FlamenWolf** [comment]
> Just found out about this fork that works with proton https://github.com/Cloudef/ModEngine2/releases/tag/2.0.0.1-proton-v3. Hopefully it can be merged.

I tried it and I can't get the mod to load still.

**Comment by Keelhauled** [comment]
I just ended up using UXM to mod. I got that fork to work somewhat but regulation.bin and some other files were not replaced so it doesn't seem fully functional anyway.

**Comment by abigrock** [comment]
Can confirm that Cloudef's fork works with latest Elden Ring update (1.08.1) and Elden Ring Reforged + Seamless Coop. Using Proton GE although probably not required.

**Comment by Pednick** [comment]
> Can confirm that Cloudef's fork works with latest Elden Ring update (1.08.1) and Elden Ring Reforged + Seamless Coop if setup according to [the instructions](https://github.com/Cloudef/ModEngine2/releases/tag/2.0.0.1-proton-v3). Using Proton GE although probably not required.

I'm using proton experimental and found that only one mod which is EBB and it's associated armor mods:
[https://www.nexusmods.com/eldenring/mods/873](url)

Start up but don't load all the way:
https://i.imgur.com/jLR8FI0.jpg[](url)

**Comment by ThePyrotechnic** [comment]
> Can confirm that Cloudef's fork works with latest Elden Ring update (1.08.1) and Elden Ring Reforged + Seamless Coop if setup according to [the instructions](https://github.com/Cloudef/ModEngine2/releases/tag/2.0.0.1-proton-v3). Using Proton GE although probably not required.

@abigrock Are you able to see the extra menu options that ERR adds to the bonfires? I wasn't able to (my notes here: https://github.com/soulsmods/ModEngine2/issues/58#issuecomment-1372641220)

**Comment by Pednick** [comment]
> > Can confirm that Cloudef's fork works with latest Elden Ring update (1.08.1) and Elden Ring Reforged + Seamless Coop if setup according to [the instructions](https://github.com/Cloudef/ModEngine2/releases/tag/2.0.0.1-proton-v3). Using Proton GE although probably not required.
> 
> @abigrock Are you able to see the extra menu options that ERR adds to the bonfires? I wasn't able to (my notes here: [#58 (comment)](https://github.com/soulsmods/ModEngine2/issues/58#issuecomment-1372641220))

Yeah I tried ERR and the text was messed up for me too otherwise if I knew what to pick it should work.

**Comment by abigrock** [comment]
@ThePyrotechnic I see this under the difficulties menu:
![image](https://user-images.githubusercontent.com/12421649/210869774-d79035ce-aa5f-495b-b1a3-d5eba41658f3.png)

It's worth noting that when I first setup the mod according to Cloudef's instructions the ERR themed menu screen wasn't showing like it does on my Windows install. For whatever reason, verifying Elden Ring's game files through steam fixed it. I didn't check to see if the mod's bonfire menus were showing correctly until after I verified the game cache and saw that the ERR themed menu was showing as expected.

**Comment by ThePyrotechnic** [comment]
> 

Okay great, so if it works for you then it definitely _can_ work with the tools we've got now. What version of Proton GE are you using?

**Comment by Pednick** [comment]
> @abigrock Okay great, so if it works for you then it definitely _can_ work with the tools we've got now. What version of Proton GE are you using?

 GE-Proton7-43

**Comment by abigrock** [comment]
@ThePyrotechnic GE_Proton7_43 from AUR

**Comment by Pednick** [comment]
> @ThePyrotechnic GE_Proton7_43 from AUR

I got mine from github.

**Comment by abigrock** [comment]
Been messing around some more and found that sometimes ERR's menu theme doesn't load and the bonfire menus show `?EventTextForTalk?`. I found that changing `load_delay` to 2500 in `mod_loader_config.ini` corrects the issue every time I've launched the game so far. Perhaps verifying the game's cache isn't necessary.

**Comment by Pednick** [comment]
> Been messing around some more and found that sometimes ERR's menu theme doesn't load and the bonfire menus show `?EventTextForTalk?`. I found that changing `load_delay` to 2500 in `mod_loader_config.ini` corrects the issue every time I've launched the game so far. Perhaps verifying the game's cache isn't necessary.

I've just tried ERR once just to see if it works. Just mainly playing pure seemless.

**Comment by DiabeticCrab** [comment]
Does anyone know how to circumvent the
 [2023-01-24 22:11:10.875] [modengine] [error] Unable to detect a supported game
error on Linux?

It refuses to work for some reason. Neither this release, nor the Proton compatible one from Cloudef.

**Comment by FahriBilici** [comment]
Hi where did you run `echo "%command%" | sed 's/start_protected_game/eldenring/' | WINEDLLOVERRIDES="dinput8.dll=n,b" sh
`. command?

**Comment by Pednick** [comment]
> Hi where did you run `echo "%command%" | sed 's/start_protected_game/eldenring/' | WINEDLLOVERRIDES="dinput8.dll=n,b" sh `. command?

I got it from here: https://github.com/Cloudef/ModEngine2/releases/tag/2.0.0.1-proton-v3

**Comment by Pednick** [comment]
> I mean I am using lutris. where should I run this command in lutris?

I dunno, I don't use lutris. Maybe ask the author of that site.

**Comment by FahriBilici** [comment]
Are you using steam deck?

> > I mean I am using lutris. where should I run this command in lutris?
> 
> I dunno, I don't use lutris. Maybe ask the author of that site.

**Comment by Pednick** [comment]
> Are you using steam deck?
> 
> > > I mean I am using lutris. where should I run this command in lutris?
> > 
> > 
> > I dunno, I don't use lutris. Maybe ask the author of that site.

Nope, I use steam with proton. When you play on steam deck, do you have launch options for elden ring? If you do, you paste that into the launch options.

**Comment by xenanthropy** [comment]
> Been messing around some more and found that sometimes ERR's menu theme doesn't load and the bonfire menus show `?EventTextForTalk?`. I found that changing `load_delay` to 2500 in `mod_loader_config.ini` corrects the issue every time I've launched the game so far. Perhaps verifying the game's cache isn't necessary.

Ah thank you for this!!! I'd been trying to get randomizer and co-op working together and kept running into a similar issue with the ?blahblah? stuff, setting it to 2500 fixed that.
